app/views/furnitures/_furniture.html.erb

```diff
@@ -1,8 +1,10 @@
 <% edit_href = nil %>
 <% edit_href = furniture.location(:edit) if furniture.configurable? %>
 <% edit_href = furniture.to_kind_class.location(:edit) if furniture.to_kind_class.has_controller_edit? %>
-
 <section id="<%= dom_id(furniture) %>" class="mt-4 relative">
+  <%- if furniture.room.hero_image&.upload.present? %>
+    <%= image_tag furniture.room.hero_image&.upload&.variant(resize_to_fill: Media::FULL_WIDTH_16_BY_9), class: "w-full" %>
+  <%- end %>
```

Comment:
Won't this make the hero image show up once per piece `Gizmo`?

Reply:
I was a bit hasty wandering through the layers. Just moved it up to the correct layer (I hope 😅).